src/Mvc/Mvc.ApiExplorer/test/DefaultApiDescriptionProviderTest.cs

```diff
@@ -1398,7 +1398,7 @@ public void GetApiDescription_ParameterDescription_SourceFromDefault_ModelBinder
         public void GetApiDescription_ParameterDescription_ComplexDTO()
         {
             // Arrange
-            var action = CreateActionDescriptor(nameof(AcceptsProductChangeDTO));
+            var action = CreateActionDescriptor(nameof(AcceptsOrderDTO));
```

Comment:
Could you leave the existing test be as-is? Do your changes result in it being different?

Reply:
ok. will undo.